lib/slack/responses/ping.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  patterns: ['!ping'],
+  types: ['ambient', 'mention'],
+
+  // TODO(karlbright): I don't like this naming, should this be fn? exec? something?
```

Comment:
That makes sense to me, works for all other options(webhooks, slash commands) too.